awx_collection/README.md

```diff
@@ -12,13 +12,11 @@
 [comment]: # (*  upon build of the collection                       *)
 [comment]: # (*******************************************************)
 
-This Ansible collection allows for easy interaction with an AWX server via Ansible playbooks.
+The AWX Ansible collection allows for easy interaction with an AWX server via Ansible playbooks.
 
-This source for this collection lives in the `awx_collection` folder inside of the
-AWX GitHub repository.
-The previous home for this collection was inside the folder [lib/ansible/modules/web_infrastructure/ansible_tower](https://github.com/ansible/ansible/tree/stable-2.9/lib/ansible/modules/web_infrastructure/ansible_tower) in the Ansible repo,
-as well as other places for the inventory plugin, module utils, and
-doc fragment.
+- Ansible Galaxy: [galaxy.ansible.com/awx/awx](https://galaxy.ansible.com/awx/awx)
+- Source: [github.com/ansible/awx/./awx_collection](https://github.com/ansible/awx/tree/devel/awx_collection)
+- Docs: [`./awx_collection/README.md`](https://github.com/ansible/awx/blob/devel/awx_collection/README.md)
```

Comment:
The `galaxy.yml` has structured data to link to the homepage and docs. I agree that a link in context here makes sense if to say "these modules originated at X but are now at Y"

Reply:
Very confusing to a new user. Why is this the first link in the document, and why is there no link to where it is now? Galaxy may display that metadata, but gh READMEs do not.

e.g. https://Shields.io has badges for .md and .rst READMEs:
- [x] gh source
- [x] docs
- [x] pypi
- [ ] ansible galaxy README badges

...

- [ ] DOC: update all of the ansible collection readmes w/ badges